modules/guides/pages/javascript-udfs/introduction.adoc

```diff
@@ -0,0 +1,47 @@
+= User-defined Functions in Javascript
+:description: Extending the {sqlpp} query language by adding your own functions written in Javascript
```

Comment:
If you're using a local playbook then you need to define it in there. It's already defined in the playbooks for production.

Reply:
Mmm. Yeah, having thought about it for a while, this isn't really what attributes should be used for anyway. The big problem is that N1QL, SQL++ or whatever it might be in the future might take an 'a' or an 'an' (and other grammarly things I haven't thought of), so every change should really be checked anyway just to make sure. And if you have to check, then I'm not sure an attribute is going to help that much in this case.

Reply:
Raised a PR on docs-site.

Reply:
👍🏾

Reply:
> The big problem is that N1QL, SQL++ or whatever it might be in the future might take an 'a' or an 'an'

Yep — I raised this in style and tone. You're right about the proper use of attributes. But if we _don't_ use the attribute, then we would have to make the change all at once, in one intense docs sprint 😬